checklists/apim_checklist.en.json

```diff
@@ -299,6 +303,7 @@
             "text": "Deploy Private Endpoints to filter incoming traffic when APIM is not deployed to a VNet.",
             "guid": "67437a28-2721-4a2c-becd-caa54c8237a5",
             "severity": "Medium",
+            "graph": "resources | where type == "microsoft.apimanagement/service" | extend compliant = (properties.virtualNetworkType == "None" and isnotnull(properties.privateEndpointConnections)) | distinct id, compliant",
```

Comment:
Double quotes in the query string break the JSON syntax, please use single quotes